src/components/molecules/Visualizer/Engine/Cesium/hooks.ts

```diff
@@ -163,6 +191,206 @@ export default ({
     }
   }, [camera, engineAPI]);
 
+  const geodsic = useMemo(():
+    | undefined
+    | { geodesicVertical: EllipsoidGeodesic; geodesicHorizontal: EllipsoidGeodesic } => {
+    const viewer = cesium.current?.cesiumElement;
+    if (
+      !viewer ||
+      !viewer.scene ||
+      viewer.isDestroyed() ||
+      !property?.cameraLimiter ||
+      !property?.cameraLimiter.cameraLimitterTargetArea
+    )
+      return undefined;
+    const ellipsoid = viewer.scene.globe.ellipsoid;
+
+    const centerPoint = Cartesian3.fromDegrees(
+      property.cameraLimiter.cameraLimitterTargetArea.lng,
+      property.cameraLimiter.cameraLimitterTargetArea.lat,
+      0,
+    );
+
+    const CartographicCenterPoint = Cartographic.fromCartesian(centerPoint);
+    const normal = Ellipsoid.WGS84.geodeticSurfaceNormal(centerPoint);
+    const east = Cartesian3.normalize(
+      Cartesian3.cross(Cartesian3.UNIT_Z, normal, new Cartesian3()),
+      new Cartesian3(),
+    );
+    const north = Cartesian3.normalize(
+      Cartesian3.cross(normal, east, new Cartesian3()),
+      new Cartesian3(),
+    );
```

Comment:
```suggestion
    const east = Cartesian3.normalize(
      Cartesian3.cross(Cartesian3.UNIT_Z, normal),
    );
    const north = Cartesian3.normalize(
      Cartesian3.cross(normal, east),
    );
```

Result arg can be omitted

Reply:
Unfortunately Not in this function, it gives an error `An argument for 'result' was not provided.`